doc/developer/debugging.rst

```diff
@@ -196,7 +196,7 @@ Run:
 
 .. prompt:: bash
 
-   docker-compose --file=docker-compose.yaml --file=docker-compose-dev.yaml up
+   docker-compose --file=docker-compose.yaml --file=docker-compose-dev.yaml up -b
 
 Open the application with on the following path: ``https://<host>/<entry_point>/dev/<interface>.html``.
```

Comment:
redundant "with on" -> suggestion: "at the following path:"

Reply:
Thanks :-)